tests/testthat/_snaps/get-polymod-population.md

```diff
@@ -27,18 +26,17 @@
       # A tibble: 21 x 2
          lower.age.limit population
                    <int>      <dbl>
-       1               0   1841420.
-       2               5   1950666.
-       3              10   2122856.
-       4              15   2323822.
-       5              20   2406141.
-       6              25   2377541.
-       7              30   2552587.
-       8              35   2982293.
-       9              40   3044427.
-      10              45   2828202.
+       1               0   1852682.
+       2               5   1968449.
+       3              10   2138897.
+       4              15   2312032.
+       5              20   2407486.
+       6              25   2423602.
+       7              30   2585137.
+       8              35   2969393.
+       9              40   3041663.
+      10              45   2809154.
       # ... with 11 more rows
```

Comment:
I think we discussed this in our last meeting - are these numbers different due to some random seed issue or something?